tor_core/initialize.py

```diff
@@ -5,13 +5,11 @@
 
 import redis
 from bugsnag.handlers import BugsnagHandler
-from raven.handlers.logging import SentryHandler
-from raven.conf import setup_logging
 from praw import Reddit
 from raven import Client
-from rocketchat_API.rocketchat import RocketChat
-from rocketchat_API.APIExceptions.RocketExceptions import RocketAuthenticationException
-from rocketchat_API.APIExceptions.RocketExceptions import RocketConnectionException
+from raven.conf import setup_logging
+from raven.handlers.logging import SentryHandler
```

Comment:
Why are we reintegrating Sentry here?

Reply:
We aren't -- see above, I just automatically rearranged the imports. Those were never removed, and I didn't want to remove them as a part of this PR because it's not relevant